### PR TITLE
[helpers] add build_packages.sh

### PR DESCRIPTION
**Pull request by sledges**
HADK document will call this script to avoid manually executing a
huge amount of commands just to e.g. take in a change in patterns.

Also, build bug can now be fixed in this script, instead of releasing
a new HADK version.

### CODE REVIEW

File: helpers/build_packages.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# droid-hal device packaging converter: from monolithic to modular
```
Comment (sledges) [inline helpers/build_packages.sh]: really? (thanks @mlehtima for this find :))

File: helpers/build_packages.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# Copyright (c) 2015 Jolla Ltd.
+# Contact: Simonas Leleiva <simonas.leleiva@jollamobile.com>
```
Comment (sledges) [inline helpers/build_packages.sh:9]: Also wrong. Here should go the proper licence preamble of dhd (BSD-3-Clause) and authors are Alin and Oleid (attribute that this code has been referenced from sfa-mer, if WTFPL exercises attribution)

Comment (alinelena) [inline helpers/build_packages.sh:9]: put whatever is the right one... i did not change it on purpose

**Comment by alinelena** [comment]
looks great!
Alin

**Comment by sledges** [comment]
Thank you for this contrib!

**Pull request merged**